### PR TITLE
Hello Microverse Project 

### PR DESCRIPTION
**Pull request by sumon766**
Setup linter
Fixed linter errors
Update PR title
Update PR description

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Ignore all the .txt files
-*.txt
\ No newline at end of file
+*.txt
+*node_modules
```
Comment (levy002) [inline .gitignore]: - [x] Kindly make the `test.md` file ignored by git so it must not be represented in the repo and it must be presented in `gitignore` file. :wink:

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @sumon766   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to add a good PR title and PR description. It makes your repo looks more professional. :wink: 

   - Example of good PR
image

  - ### How to edit PR 
      chrome-capture-2023-0-11

- [ ] Kindly make sure to solve all linter errors on your PR in order to be able to ready for merging into the main branch.

- [ ] Kindly make sure to add a README file by copying the [Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) and make sure to customize it according to your project. :wink: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (sumon766) [reply .gitignore]: @levy002 Updates have been made and submitted for re-review. Please check.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
-# hello-microverse
+Hello Microverse
+This is the first project with microverse. Includes index.html & style.css files along with linter setup.
```
Comment (iambenkis) [inline README.md]: - [ ] As requested by the previous code reviewer , kindly make sure to add a README file by copying the [Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) and make sure to customize it according to your project. :wink:

**Review by iambenkis — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @sumon766  :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Basic Structure is good 

Good job on implementing some requirements but there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @iambenkis in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,3 +1,58 @@
...
+
+No installation required. Just clone or download and open the index file in your browser
+
+<!-- AUTHORS -->
+
+## 👤 Author
+
+- GitHub: [@githubhandle](https://github.com/sumon766)
+- Twitter: [@twitterhandle](https://twitter.com/sumon766)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/sumon766)
```
Comment (franclobo) [inline README.md]: - [ ] As the last code reviewer said kindly follow the instructions of the README template by Microverse. All main headers from the 👉🏽 [README template](https://github.com/microverseinc/readme-template) should be present in your repo for professional documentation.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @sumon766, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @sumon766 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**